tests/test_config_templates/test_makefile.py

```
@@ -0,0 +1,200 @@
+"""Tests for the Makefile targets and help output using safe dry‑runs.
```

Ruff (0.14.8)

Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)